RabbitMQ/mini_lab/mini_lab_build/Dockerfile

```diff
@@ -0,0 +1,15 @@
+# Use the official Jupyter Notebook Image as the base image
+FROM jupyter/scipy-notebook:latest
+
+# Set the working directory
+WORKDIR /app
+
+# Copy and install dependencies
+COPY requirements.txt .
+RUN pip install --no-cache-dir -r requirements.txt
+
+#Set envrionmental variables
+ENV PYTHONPATH /app

```

Comment:
ENV path is missing the producer, consumer & market_watch dirs. This ensures that the solutions of candidates can be found in jpylab. Refer to internal repo for example